include/bpf/bpf.h

```diff
@@ -13,10 +13,10 @@
  *
  * @param[in] link_fd File descriptor of link to detach.
  *
- * @retval 0 The operation was successful.
- * @retval <0 An error occured, and errno was set.
+ * @return 0, on success; negative error code, otherwise (errno is also set to
+ * the error code)
```

Comment:
Should not use both `@return` and `@retval`.  Switch back to just using retval.

Reply:
Fixed.